sphinx/writers/manpage.py

```diff
@@ -23,7 +23,6 @@
 
 logger = logging.getLogger(__name__)
 
-
 class ManualPageWriter(Writer):
```

Comment:
```suggestion

class ManualPageWriter(Writer):
```

Reply:
do you maybe have a quick hint on how to run linters? I get

```
$ make
Ruff: An extremely fast Python linter.
```

with ruff version 0.3.3

Reply:
It should be `make style-check`. I don't think we have a rule to *apply* the ruff formatter though.

Reply:
Ok, nvm it was wrong

Reply:
Use ``ruff check sphinx`` or ``ruff check tests`` if you have added tests.

Reply:
And thank you because we did detect now an issue with the makefile !